src/agents/run.py

```diff
@@ -1138,6 +1138,12 @@ async def _start_streaming(
 
             streamed_result.is_complete = True
         finally:
+            if streamed_result._input_guardrails_task:
+                try:
+                    streamed_result.input_guardrail_results = await streamed_result._input_guardrails_task
+                except Exception:
+                    # Exceptions will be checked in the stream_events loop
+                    output_guardrail_results = []
```

Comment:
I don't think this output guardrail related local variable is necessary here. Instead, can you add debug-level logging for debugging what's going on during local development?

Reply:
I'm definitely open to taking some time to ensure it works as intended, I am a little limited on time today so may need to followup tomorrow or Monday if that's alright.

I already have a repro file I can use to test the before and after behavior so shouldn't be hard. I'll take a look at the unit tests and see if I can add a few for the relevant scenarios

Reply:
Thank you so much!